### DISCUSSION TITLE
ribir seems not to be published to crates.io yet

### DISCUSSION THREAD

**Discussion by zoechi** [RibirX/Ribir#260] [General]
or did I miss something?
So this instruction from https://github.com/RibirX/Ribir/blob/master/docs/getting_started/quick_start.md can not work yet.

> And next, edit the `Cargo.toml` file and add Ribir dependencies:
> 
> ```toml
> [dependencies]
> ribir = "0.0.1"
> ```
> 

What are the plans to publish a first version?

**Comment by M-Adoo** [comment]
#261 merged.

Maybe next month publish the first version, we have some basic work that needs finish.

**Reply by zoechi** [reply]
I see. I guess it was worth it to change in the docs then.
Ribir was brought to my attention in a Reddit post a few days ago. 
Many others might want to have a look and this might prevent some early confusion.

**Reply by M-Adoo** [reply]
Thank you again, what you are doing is important to us.

**Reply by zoechi** [reply]
Thank you for working on Ribir. It looks amazing already, but I only just started looking at it.

**Reply by willief** [reply]
Ribir was mentioned on [safenetforum.org](url), the community forum of the SAFE Network project, the world's first fully autonomous data and communications network.
Unfortunately the response was less than enthusiastic, precisely due to this issue. I myself hit this  brick wall immediately but can definitely see the potential. 
If you are not on crates.io just yet, can it be made available elsewhere? 

SAFE is an ambitious project, they simply want to make the internet work for users by getting rid of servers entirely. And they're doing it in Rust.  For now SAFE is entirely CLI apart from some baby GUI steps by community members. Work on a dedicated SAFE browser is on hold until the underlying code is ready for a test release. 

Please come check it out, cos we'd love to see what use we could make of Ribir

[Forum](https://safenetforum.org)     <-- latest news
[Primer](https://primer.safenetwork.org/) <-- please read this
[Company site](https://maidsafe.net) <-- interesting but needs updated

**Comment by zoechi** [comment]
@willief The doc linked to in my OP above was updated and shows how to use it from GitHub directly until it is published
